modules/appeals_api/config/schemas/10182.json

```diff
@@ -40,18 +40,19 @@
       "required": [ "boardReviewOption", "socOptIn" ]
     },
 
-
     "nodCreateVeteran": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
-        "address":             { "$ref": "#/definitions/nodCreateAddress", "minLength": 5, "maxLength": 165 },
+        "address":             { "$ref": "#/definitions/nodCreateAddress" },
```

Comment:
I know I asked you to take this out but it _might_ be showing up in the documentation as a way of indicating to the consumer that the concatenated length has limits. I think we also added a `description` to the address somewhere that indicated these limits as well, so as long as there's _some_ indication in the documentation of the concatenated limit, it's all good.

Reply:
K I'll check!

Reply:
Description exists: `Address of the Veteran - not required if Veteran is homeless. Cannot exceed 165 characters when all fields are concatenated.`